tests/test_main.py

```diff
@@ -2193,25 +2193,28 @@ class Model(BaseModel):
 
 
 def test_protected_namespace_default():
-    with pytest.raises(NameError, match='Field "model_x" has conflict with protected namespace "model_"'):
+    with pytest.raises(NameError, match='Field "model_prefixed_field" has conflict with protected namespace "model_"'):
 
         class Model(BaseModel):
-            model_x: str
+            model_prefixed_field: str
 
 
 def test_custom_protected_namespace():
-    with pytest.raises(NameError, match='Field "test_x" has conflict with protected namespace "test_"'):
+    with pytest.raises(NameError, match='Field "test_field" has conflict with protected namespace "test_"'):
 
         class Model(BaseModel):
-            test_x: str
+            model_prefixed_field: str
```

Comment:
I think it worth adding a comment here about why this field is present here and that it should be defined before the protected one for test to work correctly.